src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/repositories/TelevisionRepository.java

```diff
@@ -0,0 +1,31 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.repositories;
+
+import com.novi.backend_spring_boot_tech_it_easy_controller.models.Television;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import java.util.List;
+
+
+@Repository
```

Comment:
Deze annotatie is hier niet nodig. We extenden namelijk al van JpaRepository en daar wordt dit al geregeld.